target/scripts/startup/fixes-stack.sh

```diff
@@ -21,7 +21,7 @@ function _fix_var_mail_permissions
   if find /var/mail -maxdepth 3 -a \( \! -user 5000 -o \! -group 5000 \) | read -r
   then
     _notify 'inf' 'Fixing /var/mail permissions'
-    chown -R 5000:5000 /var/mail || _shutdown 'Failed to fix /var/mail permissions'
+    chown -R 5000:5000 /var/mail || _notify 'err' 'Failed to fix /var/mail permissions'
```

Comment:
As mentioned, this line (and if you ask me, only this line,) should remain as a fatal error.